docs/reportData.html

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title: Reports Data API
+---
+
+<p>Campaign provides an API to fetch reports data. Just like all APIs in the SDK, it's been wrapped into a function and will return a XML or JSON object based on the requested representation</p>
+
+<p>Function Definition</p>
+<pre class="code">
+client.getReport = (callContext: any, representation: string) : Promise<any>
+</pre>
+<p>callContext</p>
+<pre class="code">
+{
+  reportName: string<report Name>,
+  context: <"selection"|"global">,
+  selection: string<Delivery Id>,
+  selectionCount: number<count of deliveries>,
+  formData: any
+}
+</pre>
+
+<p>Usage</p>
+<pre class="code">
+const report = await client.getReport({
+  reportName: "throughput",
+  context: "selection",
+  selection: "12133",
+  selectionCount: 1
+});
+console.log(JSON.stringify(report));
+</pre>
+<p>Usage with form data</p>
+<pre class="code">
+const report = await client.getReport({
+  reportName: "throughput",
+  context: "selection",
+  selection: "12133",
+  selectionCount: 1,
+  formData: {
+    vars_opens: 1,
+    userAction: "next",
+    ctx: {
+      _context: 'selection',
+      _reportContext: 'throughput',
+      _hasFilter: 'false',
```

Comment:
In other examples, I've tried to use strongly typed data, i.e. false instead of "false", etc. I'm not really convinced in passing everything as string. I think documentation will be use as a n example / best practices and should use typed values (if it actually works of course :-)